src/operators.jl

```diff
@@ -196,21 +207,24 @@ for op in ops_2_to_1_fsc
     SparseConnectivityTracer.is_crossder_zero_global(::T)      = false
 end
 
+# gradient of x/y: [1/y -x/y²]
+SparseConnectivityTracer.is_firstder_arg2_zero_local(::typeof(Base.:/), x, y) = iszero(x)
```

Comment:
it can be just `::typeof(/)`

Reply:
`:/` is a bit easier to find and jump to than `/`, so I'll leave this.